generated/nirfmxspecan/nirfmxspecan.proto

```diff
@@ -5350,12 +5350,12 @@ message IDPDCfgReferenceWaveformRequest {
   double dx = 4;
   repeated nidevice_grpc.NIComplexNumberF32 reference_waveform = 5;
   oneof idle_duration_present_enum {
-    IdpdReferenceWaveformIdleDurationPresent idle_duration_present = 6;
-    int32 idle_duration_present_raw = 7;
+    int32 idle_duration_present_raw = 6;
```

Comment:
For clarity on these two, this is fixing the breaking change that we missed in https://github.com/ni/grpc-device/pull/1029/files#diff-698499203913617af06118a7f6c4ff2607d89dc18eeacdd7ae3ac538f09e264b

That change hadn't gone into the previous release so this is fixing it before it gets into this release.

Before (in the previous release), `idle_duration_present` was field 6 and `signal_type` was field 7. So this change is avoiding a binary breaking change getting into this release.